lib/tasks/refinery/elasticsearch.rake

```diff
@@ -15,6 +15,9 @@ namespace :refinery do
 
     desc 'Recreate Elasticsearch index'
     task recreate: :setup_logger do
+      # Use the right language
+      ::I18n.locale = ::I18n.default_locale
```

Comment:
What happens if i have multiple locales ? Should not we loop each locale ? What do you think @datenimperator ?